src/openepd/api/base_sync_client.py

```diff
@@ -151,6 +151,7 @@ class SyncHttpClient:
 
     HTTP_DATE_TIME_FORMAT = "%a, %d %b %Y %H:%M:%S %Z"
     DEFAULT_RETRY_INTERVAL_SEC = 10
+    TIMEOUT_SEC_DEFAULT = 3 * 60
```

Comment:
I actually did this in my branch (will go with error handling fixes). Also just 3 * 60 doesn't look good. If you set such a big timeout we have to separate connection timeout and data timeouts, otherwise it will wait 3 minutes for establishing connection in case of network failures as it happened yesterday.

Reply:
I will drop this commit then.